### DISCUSSION TITLE
Model for Image Classification

### DISCUSSION THREAD

**Discussion by TBNRGarret** [mrdbourke/pytorch-deep-learning#1284] [Q&A]
Hello everyone, I'm making a model for image classification, the problem is I have about 350 images with the size of 2000 x 1000, can anyone recommend me some good pre-trained models for this or what should I do with the data.

**Comment by Haji-Suleman** [comment]
from torchvision import transforms

transform = transforms.Compose([
    transforms.Resize((224,224)),
    transforms.ToTensor(),
    transforms.Normalize(mean=[0.485,0.456,0.406], std=[0.229,0.224,0.225])
])
This reduces memory and speeds up training
